types/generated-snapshot/2021-11-03/index.d.ts

```diff
@@ -467,7 +467,7 @@ interface Cloudflare {
 }
 interface DurableObject {
   fetch(request: Request): Response | Promise<Response>;
-  alarm?(): void | Promise<void>;
+  alarm?(alarmInfo?: AlarmInvocationInfo): void | Promise<void>;
```

Comment:
The changes to `types/generated-snapshot/*` were all generated by running the command recommended by a CI error message:
```
  The generated output of @cloudflare/workers-types has been changed by this
  PR. If this is intentional, run:
  bazel build //types && rm -rf types/generated-snapshot && cp -r bazel-bin/types/definitions types/generated-snapshot
```
I assume it's doing the right thing by regenerating these snapshot files, but I'm not familiar with the intent.